CHEATSHEET.md

```diff
@@ -215,3 +219,33 @@ cy.createPage();
 cy.createPerson();
 cy.logout();
 ```
+
+### Delete a piece of content from the node page
+
+```markdown
```

Comment:
We need a visit command here first.

// Visit a node
cy.visit('node/[nodeID]')